packages/hop-node/src/watchers/BondWithdrawalWatcher.ts

```diff
@@ -121,13 +112,28 @@ class BondWithdrawalWatcher extends BaseWatcher {
       return
     }
 
-    const availableCredit = await destBridge.getAvailableCredit()
+    let availableCredit = await destBridge.getAvailableCredit()
+    const includePendingAmount = [Chain.Optimism, Chain.Arbitrum].includes(destinationChain)
```

Comment:
This pendingAmount check should only apply when the destination chain is L1 and the source chain is a chain whose transfer root is being bonded. This is because, for example, a bonding of a withrdawal from o -> e would require `100` credit and then the bonding of that transfer root would require another `100` of credit, meaning the bonder would need `200` credit for a simple bond withdrawal of `100`. This double counting will be alleviated in v2 with `bondTransferRootAndSettle`.

Because of that, this we need to check here if source = opt/arb and destination = eth

Sorry, I missed this the first time around.

Edit: it looks like the check is correct in the SDK, so only this would need to be updated.

Reply:
you're completely right. [Fixed](https://github.com/hop-protocol/hop/pull/146/files#diff-0c3ab79e1932de7478f738aa7851502fe92140eb706c6d86cb29b6dd831c94aeR117)